fem/fespace.hpp

```diff
@@ -1371,12 +1458,22 @@ class FiniteElementSpace
        Safe to call multiple times, does nothing if space already up to date. */
    virtual void Update(bool want_transform = true);
 
+   /** P-refine and update the space. If @a want_transfer, also maintain the old
+       space and a transfer operator accessible by GetPrefUpdateOperator(). */
+   virtual void PRefineAndUpdate(const Array<pRefinement> & refs,
+                                 bool want_transfer = true);
+
+   /// Return true iff p-refinement is supported in this space.
```

Comment:
Maybe summarize here the cases that are supported?

Reply:
Done in fc63c5e90b5ace705c64319123d3715d1dbe250f